### PR TITLE
Rollup reports

### PR DESCRIPTION
**Pull request by bethbrains**
# Critical Changes

# Changes
- New reports:
1. Contacts with 3 Consecutive Absences
2. Contacts Absent for Last Svc Delivery
3. Contacts w/o Recent Service Deliveries

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
- ~~Any net new LWC work has Sa11y tests & 50% or above lines JEST test coverage~~
- ~~CRUD/FLS is enforced in Apex~~
- ~~Permission sets are updated to account for CRUD|FLS|Tab|Classes~~
- ~~Field sets are updated to account for new fields~~
- [x] UX approval or UX not necessary
- [x] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: W-0000000: Work Name (https://github.com/SalesforceFoundation/PMM/pull/303)
- [ ] All **acceptance criteria** have been met
    - [x] Developer
    - [ ] Code Reviewer
    - [ ] QA
- [x] PR contains draft release notes
- [ ] QE story level testing completed

### CODE REVIEW

**Comment by bethbrains** [comment]
[W-8789464](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B00000094eYTIAY/view)
[W-8766472](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B00000094CC3IAM/view)
[W-8766558](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B00000094CIQIA2/view)

**Comment by bmuniswamy** [comment]
@bethbrains, I see the same above behavior in the `Contacts with 3 Consecutive Absences` report too. I think it has to do something with the cross filter.

**Review by bmuniswamy — COMMENTED** [review]
I have a couple of questions.

File: force-app/main/default/reports/ProgramManagementReports/Contacts_wo_Recent_Service_Deliveries_2SV.report-meta.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            <value>LAST_N_DAYS:14</value>
+        </criteriaItems>
+        <criteriaItems>
+            <column>Contact$LastServiceDate__c</column>
+            <columnToColumn>false</columnToColumn>
+            <isUnlocked>true</isUnlocked>
+            <operator>equals</operator>
+            <value/>
+        </criteriaItems>
+        <language>en_US</language>
```
Comment (jjbennett) [inline force-app/main/default/reports/ProgramManagementReports/Contacts_wo_Recent_Service_Deliveries_2SV.report-meta.xml:50]: I believe we need to remove this flag for orgs that do not have translations enabled.

**Review by jjbennett — CHANGES_REQUESTED** [review]

**Review by jjbennett — APPROVED** [review]

**Review by jessicahaas1 — COMMENTED** [review]
Looks good

**Pull request merged**